### ISSUE TITLE
[bug] Cut/Copy functionality

### ISSUE THREAD

**Issue by csholmq** [tldraw/tldraw#385]
I wouldn't call this a bug really but it got me confused. 

In the main menu
![image](https://user-images.githubusercontent.com/1297882/143505096-18f343db-0417-43dc-8cd9-665a3a776b7f.png)
you have both `Cut` and `Copy`. They're also both clickable regardless if an object is selected or not. Not sure what the expected behaviour is there.

Whereas in the right click menu
![image](https://user-images.githubusercontent.com/1297882/143505070-46d28fb8-c5b2-4575-bdcc-a5800bdfce59.png)
you have only `Copy`, but no `Cut`? Also missing the `Copy as JSON` for completeness.

So to summarize:
* In main menu, `Copy` and `Cut` have unclear behaviour when no object is selected. Disable?
* Missing `Cut` in right click menu. Add?
* `Copy as JSON` missing in right click menu. Add? Or remove from main menu as all objects are JSON when pasted outside of TLDR.

**Comment by BRO3886** [comment]
I think copy as json is available on the context menu (right click) when in debug mode, is it really required in prod, on the right click menu?

**Comment by csholmq** [comment]
@BRO3886 this is from the public web client. Not sure which mode that is in.

But since Copy seems to paste as JSON outside of tldraw I agree with you that it makes sense to remove that alternative from the main menu rather than adding it to the right click menu.

**Comment by BRO3886** [comment]
Right, I'll leave that upto @steveruizok, opened a PR for other stuff:
* added cut option on right click menu
* disabled copy and cut options when no item is selected

**Comment by steveruizok** [comment]
This issue is closed by #393!

**Comment by steveruizok** [comment]
> But since Copy seems to paste as JSON outside of tldraw I agree with you that it makes sense to remove that alternative from the main menu rather than adding it to the right click menu.

I'm pretty sure there's a way to save this to the clipboard without having this happen, but I haven't looked into it. Anyone here with experience?

**Comment by BRO3886** [comment]
default copy as JSON is fine, I think copy as PNG or SVG can be added
